concordia/documents.py

```diff
@@ -176,6 +183,13 @@ class AssetDocument(DocType):
         }
     )
 
+    submission_count = fields.IntegerField()
+
+    def prepare_submission_count(self, instance):
+        return Transcription.objects.filter(
+            asset=instance, submitted__isnull=True
+        ).count()
```

Comment:
Do we need to filter this on uniqueness in case there's a submit->request changes cycle?